stories/organisms/AutosuggestDeprecated.js

```diff
@@ -0,0 +1,184 @@
+// import * as React from 'react';
```

Comment:
Is this file still needed?

Reply:
I was wondering about that. I was thinking a bit to keep it for internal reference. But perhaps that is not needed

Reply:
removed it